xarray/tests/test_backends.py

```diff
@@ -608,10 +608,6 @@ def test_orthogonal_indexing(self):
             actual = on_disk.isel(**indexers)
             assert_identical(expected, actual)
 
-    @pytest.mark.xfail(
-        not has_dask,
-        reason="the code for indexing without dask handles negative steps in slices incorrectly",
-    )
```

Comment:
Let's keep this xfail specifically for the case where it fails. It's a good remind that something is missing from Zarr + xarray if not using Dask.